make/Main.gmk

```diff
@@ -1122,7 +1136,9 @@ endif
 docs-image: docs-jdk
 
 # This target builds the test image
-test-image: prepare-test-image test-image-jdk-jtreg-native test-image-demos-jdk
+test-image: prepare-test-image test-image-jdk-jtreg-native \
+    test-image-demos-jdk test-image-libtest-jtreg-native \
+    test-image-lib-native
```

Comment:
Are we getting rid of OpenJ9 patches?

Reply:
This looks to have come straight from openjdk.

Reply:
https://github.com/ibmruntimes/openj9-openjdk-jdk15/commit/1a0d15411f5042be9fb78ede85c4a272d175a544#diff-db3facd0a5ba03518baca7e9a30465068f70b3ea29dd3988f6dbaf3b246280d1
Note the target `test-image-libtest-jtreg-native`.

Reply:
Not intentionally.  I don't see any OpenJ9 patch to these lines. The change here is reformatting, plus adding `test-image-lib-native` which comes from the OpenJDK 15.0.2 changes. The next few lines after this, which were added for OpenJ9, will discard `test-image-lib-native`, but I'm not sure it's needed for OpenJ9 anyway. The PR build did fail, but it looks unrelated.

Reply:
I see it now, updating ...

Reply:
Updated